lib/exercism/team_stream_filters.rb

```diff
@@ -72,7 +72,7 @@ def sql
     # rubocop:disable Metrics/MethodLength
     def views_sql
       <<-SQL
-      SELECT 1 AS id, COUNT(views.id) AS total
+      SELECT '#{team_slug}' AS id, COUNT(1) AS total
```

Comment:
Thinking if this can be dangerous... Maybe not since this code is only executed if the team is valid, but still...